src/libraries/System.Data.Common/tests/System/Data/SqlTypes/SqlStringSortingTest.cs

```diff
@@ -27,8 +27,9 @@ public static class SqlStringSortingTest
 
         private static readonly string[,] s_specialMatchingString = new string[4, 2] {{"Lorem ipsum dolor sit amet", "\uFF2C\uFF4F\uFF52\uFF45\uFF4D\u3000\uFF49\uFF50\uFF53\uFF55\uFF4D\u3000\uFF44\uFF4F\uFF4C\uFF4F\uFF52\u3000\uFF53\uFF49\uFF54\u3000\uFF41\uFF4D\uFF45\uFF54"},
                                                                          {"\u304B\u305F\u304B\u306A", "\u30AB\u30BF\u30AB\u30CA"},
-                                                                         {"\uFF8C\uFF67\uFF7D\uFF9E\uFF65\uFF77\uFF9E\uFF80\uFF70", "\u30D5\u30A1\u30BA\u30FB\u30AE\u30BF\u30FC"},
-                                                                         {"engine", "eNGine"}};
+                                                                         {"engine", "eNGine"},
```

Comment:
any reason for this re-order?

Reply:
You'll know it when you review the rest of this file :-)

Reply:
ok, nvm, understand now as I read the code further